crates/autopilot/src/arguments.rs

```diff
@@ -22,17 +22,9 @@ pub struct Arguments {
     #[clap(flatten)]
     pub price_estimation: price_estimation::Arguments,
 
-    /// Address of the ethflow contract
-    #[clap(
-        long,
-        env,
-        default_value = "0x76aAf674848311C7F21fc691B0b952f016dA49F3"
-    )]
-    pub ethflow_contract: H160,
-
-    // Feature flag for ethflow.
+    /// Address of the ethflow contract. If not specified, eth-flow orders are disabled.
     #[clap(long, env)]
-    pub enable_ethflow_orders: bool,
+    pub ethflow_contract: Option<H160>,
```

Comment:
I'm interested in why we don't use the `::deployed()` machinery (i.e. include the EthFlow contract address in `contracts/build.rs` to avoid having to pass runtime parameters.

The downside being that you would need to re-build to use the new contract, but this is already the case for most of our contracts.

Reply:
The plan is to use different contracts for barn and prod: otherwise the prod/barn autopiliot would be picking up barn/prod orders but mark them as invalid in the explorer (as there is no corresponding quote id).

Reply:
Riiiiight! I had forgotten. Sounds good to me then.